### ISSUE TITLE
ARI-A Computation uses only GT Object Masks

### ISSUE THREAD

**Issue by ManuelTraub** [jinyangyuan/air-unofficial#1]
In the code, it appears that the ARI-A algorithm, which should utilize all pixels in the image, only contains ground-truth masks for objects:

```
segments_obj = segments[:, :-1]
# ARI
segments_obj_sel = segments_obj if self.seg_overlap else segments_obj * (1 - overlaps)
ari_all = self.compute_ari(segments_obj_sel, mask_oh_all)
ari_obj = self.compute_ari(segments_obj_sel, mask_oh_obj)
 ```
https://github.com/jinyangyuan/air-unofficial/blob/59ce7adc023be8876a0045b0c53982498fbfaf1e/src/model.py#L175C1-L175C1

For context, segments are initialized here:
```
segments = data['segment'][:, None, None].long()
scatter_shape = [segments.shape[0], segments.max() + 1, *segments.shape[2:]]
segments = torch.zeros(scatter_shape, device=segments.device).scatter_(1, segments, 1)
```
https://github.com/jinyangyuan/air-unofficial/blob/59ce7adc023be8876a0045b0c53982498fbfaf1e/src/model.py#L31C9-L33C95

In the dataset "segment" contains all object masks + the background mask. The Background mask has the highest index and therefore is the last mask in the one hot encoded segments. So the background mask is correctly removed for ari_obj, but should be included in the ari_all computation.
